kaspresso/src/main/kotlin/com/kaspersky/kaspresso/files/dirs/AllureDirsProvider.kt

```diff
@@ -0,0 +1,55 @@
+package com.kaspersky.kaspresso.files.dirs
+
+import android.annotation.SuppressLint
+import android.app.Instrumentation
+import android.content.Context
+import android.os.Build
+import android.os.Environment
+import com.kaspersky.kaspresso.files.resources.ResourcesRootDirsProvider
+import com.kaspersky.kaspresso.instrumental.InstrumentalDependencyProvider
+import com.kaspersky.kaspresso.internal.extensions.other.createDirIfNeeded
+import com.kaspersky.kaspresso.internal.extensions.other.createFileIfNeeded
+import java.io.File
+
+class AllureDirsProvider(
```

Comment:
All allure related stuff should be in `allure-support`

Reply:
Done